new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleOperatorValidation.java

```diff
@@ -59,18 +65,37 @@ public void testInstallingOperator() {
     // the kubernetes deployment.  this will complete quickly, and will either be
     // successful or not.
 
-    String opns = "opns1";
-    OperatorParams opParams
-        = new OperatorParams().image("weblogic-kubernetes-operator:test_itsimpleoperator")
-          .domainNamespaces(Arrays.asList("domainns1", "domainns2"))
-          .serviceAccount("sa-opns1");
-    boolean success = installOperator("weblogic-operator", opns, opParams);
+    // get a new unique namespace
+    final String namespace = assertDoesNotThrow(TestActions::createUniqueNamespace,
+        "Failed to create unique namespace due to ApiException");
+    logger.info(String.format("Got a new namespace called %s", namespace));
 
-    // we can use a standard JUnit assertion to check on the result
-    assertEquals(true, success, "There MUST be a descriptive message here");
+    // Create a service account for the unique namespace
+    final String serviceAccountName = namespace + "-sa";
+    assertDoesNotThrow(
```

Comment:
wow, that's a bit awkward.  is there an assertJ assertion we can use instead of this one, that makes the code a bit nicer?

Reply:
will see ...